nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/DeleteHBaseRow.java

```diff
@@ -103,6 +103,17 @@
             .defaultValue("UTF-8")
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
+    static final PropertyDescriptor VISIBLITY_LABEL = new PropertyDescriptor.Builder()
+            .name("delete-visibility-label")
+            .displayName("Visibility Label")
+            .description("If visibility labels are enabled, a row cannot be deleted without supplying its visibility label(s) in the delete " +
+                    "request. Note: this visibility label will be applied to all cells within the row that is specified. If some cells have " +
+                    "different visibility labels, they will not be deleted. When that happens, the failure to delete will be considered a success " +
+                    "because HBase does not report it as a failure.")
+            .required(false)
```

Comment:
Since this is not required and always valid, a user could force the property to be set with an empty string. Is that OK? It means visibility would be equal to "". Does that make sense in some cases? Wouldn't be better to add a non-empty string validator?